frc46_factory_token/src/lib.rs

```diff
@@ -10,18 +13,21 @@ fn token_invoke(method_num: u64, params: u32) -> Result<u32, RuntimeError> {
     match_method!(method_num, {
         "Constructor" => {
             let params = deserialize_params(params);
-            construct_token(params)
```

Comment:
nit: if the runtime is being instantiated in every match branch, perhaps move to before the `match_method!`